Core/src/org/sleuthkit/autopsy/casemodule/MultiUserCase.java

```diff
@@ -110,34 +121,81 @@ Date getLastAccessedDate() {
      * @return See CaseStatus enum definition.
      */
     CaseStatus getStatus() {
-        if (AutoIngestAlertFile.exists(caseDirectoryPath)) {
+        if(caseDirectoryPath.resolve("autoingest.alert").toFile().exists()) {
             return CaseStatus.ALERT;
         } else {
             return CaseStatus.OK;
         }
     }
 
     /**
+     * Gets the case metadata from a case directory path.
+     *
+     * @param caseDirectoryPath The case directory path.
+     *
+     * @return Case metadata.
+     */
+    static CaseMetadata getCaseMetadataFromCaseDirectoryPath(Path caseDirectoryPath) throws CaseMetadata.CaseMetadataException {
+        CaseMetadata caseMetadata = null;
+        
+        File directory = new File(caseDirectoryPath.toString());
+        if (directory.isDirectory()) {
+            String fileNamePrefix = directory.getName();
+            if (TimeStampUtils.endsWithTimeStamp(fileNamePrefix)) {
+                fileNamePrefix = fileNamePrefix.substring(0, fileNamePrefix.length() - TimeStampUtils.getTimeStampLength());
+            }
+            
+            /*
+             * Attempt to open an AUT file that has the folder name without a
+             * time stamp.
+             */
+            File autFile = new File(directory + "/" + fileNamePrefix + CaseMetadata.getFileExtension());
+            
+            /*
+             * If the AUT file doesn't exist, attempt to find an AUT file via a
+             * directory scan.
+             */
+            if(!autFile.isFile()) {
+                for (File file : directory.listFiles()) {
+                    if (file.getName().toLowerCase().endsWith(CaseMetadata.getFileExtension()) && file.isFile()) {
+                        autFile = file;
+                        break;
+                    }
+                }
+            }
+            
+            /*
+             * If the AUT file has been found, grab the case metadata.
+             */
+            if(autFile.isFile()) {
+                caseMetadata = new CaseMetadata(Paths.get(autFile.getAbsolutePath()));
+            }
+        }
+        
+        return caseMetadata;
```

Comment:
This method can return null (not documented), and your use of it does not check the return value.  Personally, I would define a MultiUserCaseException class, which keeps the use of CaseMetadata encapsulated instead of coupling all clients of this class to CaseMetadata  via CaseMetadataException. Then, if this method throws, the error can just propagate and there is no use of the error prone use of a special return value (null) to indicate an error condition. After all, if the case metadata file cannot be found, the case cannot be opened.

Also, a method like this should be private. Private access should be the default for everything.